### PR TITLE
Tests - Refine test cases for microbenchmark

### PR DESCRIPTION
**Pull request by abuccts**
__Description__

Refine test cases for microbenchmark:
* Refine test fixture, add BenchmarkTestCase class.
* Refine test data.
* Resolve no numa issue for test_ib_loopback_util case.

### CODE REVIEW

File: tests/helper/testcase.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    def createMockEnvs(self, envs=None):
+        """Create mock envs for tests.
+
+        Args:
+            envs (dict, optional): Environment variables to be mocked.
+                Defaults to None and will mock SB_MICRO_PATH to temp directory.
+        """
+        if not envs:
+            envs = {'SB_MICRO_PATH': self._tmp_dir}
+        print(envs)
```
Comment (yzygitzh) [inline tests/helper/testcase.py:61]: Delete

**Review by yzygitzh — CHANGES_REQUESTED** [review]

File: tests/helper/testcase.py (hunk 1 above)
Comment (abuccts) [reply tests/helper/testcase.py]: deleted

File: tests/benchmarks/micro_benchmarks/test_ib_loopback_performance.py [hunk 2]
```diff
@@ -6,113 +6,42 @@
...
+        super().setUpClass()
+        cls.createMockEnvs(cls)
+        cls.createMockFiles(cls, ['bin/run_perftest_loopback'])
 
     def test_ib_loopback_util(self):
         """Test util functions 'get_numa_cores' and 'get_free_port' used in ib-loopback benchmark."""
         port = network.get_free_port()
         assert (isinstance(port, numbers.Number))
         numa_cores = ib_loopback_performance.get_numa_cores(0)
+        if numa_cores is None:
```
Comment (cp5555) [inline tests/benchmarks/micro_benchmarks/test_ib_loopback_performance.py:32]: Is it a bug fix?

**Review by cp5555 — APPROVED** [review]

File: tests/benchmarks/micro_benchmarks/test_ib_loopback_performance.py (hunk 2 above)
Comment (abuccts) [reply tests/benchmarks/micro_benchmarks/test_ib_loopback_performance.py:32]: yes, it will fail on no numa system like wsl2

**Review by yzygitzh — APPROVED** [review]

**Pull request merged**